indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -412,6 +417,10 @@
   (let [service-associations (meta-db/get-associations-for-service context concept)]
     (get-elastic-version-with-associations context concept {:service-associations service-associations})))
 
+(defmethod get-elastic-version :subscription
```

Comment:
This function is already defined in `elasticsearch` namespace above. why is it defined here again?

Reply:
Good question! This really threw me off because I had the one defined in elasticsearch namespace first, and it didn't work because that one is not the one being used, when you don't have the associations.  These are the two defmulti defined in different name spaces but sharing the same name!  I could remove the other one, but I thought I would just leave it there because in the future, if somehow we need to association subscription with collection, it will be needed.

Reply:
Removed the one in elasticsearch.clj.

Reply:
Can't remove it. it's being used too, and the default one in elasticsearch.clj doesn't work either.